runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -172,32 +174,20 @@ public void onGrandExchangeOfferChanged(GrandExchangeOfferChanged offerEvent)
 		boolean shouldStack = offerItem.isStackable() || offer.getTotalQuantity() > 1;
 		BufferedImage itemImage = itemManager.getImage(offer.getItemId(), offer.getTotalQuantity(), shouldStack);
 		SwingUtilities.invokeLater(() -> panel.getOffersPanel().updateOffer(offerItem, itemImage, offerEvent.getOffer(), offerEvent.getSlot()));
-		this.queueNotification(offerItem, offerEvent.getOffer(), offerEvent.getSlot());
 	}
 
-	private void queueNotification(ItemComposition offerItem, GrandExchangeOffer newOffer, int slot)
+	@Subscribe
+	public void onChatMessage(ChatMessage event)
 	{
-		if (!this.config.enableNotifications())
+		if (!this.config.enableNotifications() || event.getType() != ChatMessageType.SERVER)
 		{
 			return;
 		}
 
-		// Queue a notification
-		this.notificationHandler.queueNotification(slot, offerItem, newOffer);
-	}
-
-	@Subscribe
-	public void onTick(GameTick tick)
-	{
-		// Send a notification is the handler and a notification are available
-		if (this.notificationHandler.canSendNotification())
+		Matcher match = geString.matcher(event.getMessage());
```

Comment:
You can just check contains and use Text.removeTags

Reply:
Replaced